scripts/generate-pr-body.sh

```diff
@@ -1,21 +1,77 @@
+    echo "English documentation updates up to $(sync_commit_human_readable_id) ($(today_utc))"
 }
 
+# shellcheck disable=SC2016 # Any Backticks here are markdown syntax not Bash expressions
 function pr_body {
-    echo "This PR was automatically generated."
+    echo "This is an automatically-generated sync PR to bring this translation repository up to date with the state of the English documentation as of $(today_utc) (commit $(sync_commit_link))."
     echo
-    git status --short |
-        grep "docs" |
-        awk '{print "- [ ] ["$2"](https://github.com/ethereum/solidity/tree/develop/"$2")"}'
+    echo "### How to work on a sync PR"
+    echo "#### Resolve conflicts and translate newly added text"
+    echo "- The PR includes all documentation changes from the main repository since the last time a sync PR was merged. " \
+         "If this translation repository is fully caught up with the English version, translate any newly added English text you see here by pushing more commits to the PR. " \
+         "However, if the translation is incomplete, you may prefer to leave the text added in this PR and add it to your translation checklist to handle at a later time."
+    echo "- Scan the PR for merge conflict markers. " \
+         "If there were changes in the English text that has already been translated, the PR will contain merge conflicts that need to be resolved:"
+    echo '    ```diff'
+    echo '    <<<<<<< HEAD'
+    echo '        El valor más grande representable por el tipo ``T``.'
+    echo '    ======='
+    echo '        The smallest value representable by type ``T``.'
+    echo '    >>>>>>> 800088e38b5835ebdc71e9ba5299a70a5accd7c2'
+    echo '    ```'
+    echo "    The top part of the conflict is the current translation (corresponding to the old English text), the bottom shows the new English text. " \
+         "To solve the conflict simply translate the new text, possibly reusing parts of the current translation. " \
+         "**After doing so, do not forget to remove the conflict markers.**"
+    echo "- You may get conflicts also if there were structual changes that did not affect the meaning of the text and therefore do not require retranslation. " \
+         "For example when text is moved from one page to another, you will find matching conflicts in two places and the solution is to move the translation to the new spot. " \
+         "If only whitespace changed, it may even seem like there was no change at all but if there is a conflict, there is always a reason, however trivial it may be." \
+         "Be careful, though, because there is a possibility that the text was both moved **and** modified. "
+    echo
+    echo "#### Work on one sync PR at a time"
+    echo "- Sync PRs are produced by the translation bot in regular intervals as long as there are changes in the English documentation. " \
+         "You will not lose any updates by closing this PR. " \
+         "The next sync PR will also include them."
```

Comment:
```suggestion
         "The next sync PR will also include them. The latest sync PR will always include all changes that need to be done. If you haven't worked on any Sync PR yet and there are several open Sync PRs in your repo, choose the latest (newest) one to get started."
```

Reply:
Done. Here's an updated PR: https://github.com/solidity-docs/de-german/pull/13.

If you're fine with the changes, you can merge the PR and today's run will already include the new description :)